CleverRaven/Cataclysm-DDA
# Add roofs to mil base

#### Summary
None



#### Purpose of change
Add missing roofs to military base, including ones that relied on roof magic.



#### Describe the solution
- Add roofs to containers, guard towers, tents, and a roof access shack.
- Removed a tent that didn't show up due to it being squashed by a car.



#### Describe alternatives you've considered



#### Testing
- Debug spawn and examine one 4*4 section.
- Make the appropriate adjustments.
- Restart and check the results.
- Repeat until it is correct.
- Repeat the above for each of the 4*4 sections of the map.



#### Additional context
I'm not sure it's used, but it's marked as globally unique, so I may just never have come across it (not the first thing I've never encountered).
